Example/example/SceneDelegate.swift

```diff
@@ -0,0 +1,127 @@
+        positionCameraRandomlyButton.layer.cornerRadius = 8
+        self.viewController.view.addSubview(positionCameraRandomlyButton)
+        NSLayoutConstraint.activate([
+            positionCameraRandomlyButton.trailingAnchor.constraint(equalTo: self.viewController.view.layoutMarginsGuide.trailingAnchor),
+            positionCameraRandomlyButton.centerYAnchor.constraint(equalTo: self.viewController.view.centerYAnchor),
+            positionCameraRandomlyButton.widthAnchor.constraint(equalTo: positionCameraRandomlyButton.heightAnchor),
+            positionCameraRandomlyButton.widthAnchor.constraint(equalToConstant: 44)
+        ])
+        
+        self.startButton.translatesAutoresizingMaskIntoConstraints = false
+        self.startButton.setImage(UIImage(systemName: "play.fill"), for: .normal)
+        self.startButton.addTarget(self, action: #selector(startButtonTapped), for: .touchUpInside)
+        self.startButton.backgroundColor = .white
+        self.startButton.layer.cornerRadius = 8
+        self.viewController.view.addSubview(self.startButton)
+        NSLayoutConstraint.activate([
+            self.startButton.trailingAnchor.constraint(equalTo: self.viewController.view.layoutMarginsGuide.trailingAnchor),
+            positionCameraRandomlyButton.bottomAnchor.constraint(equalTo: self.startButton.topAnchor, constant: -12),
+            self.startButton.widthAnchor.constraint(equalTo: self.startButton.heightAnchor),
+            self.startButton.widthAnchor.constraint(equalToConstant: 44)
+        ])
+    }
+}
+
+extension SceneDelegate: NavigationViewControllerDelegate {
+    func navigationViewControllerDidFinish(_ navigationViewController: NavigationViewController) {
+        navigationViewController.endNavigation()
+    }
+}
+
+// MARK: - Private
+
+private extension SceneDelegate {
+    func startNavigation(for waypoints: [Waypoint]) {
+        let options = NavigationRouteOptions(waypoints: waypoints, profileIdentifier: .automobileAvoidingTraffic)
+        options.shapeFormat = .polyline6
+        options.distanceMeasurementSystem = .metric
+        options.attributeOptions = []
+        
+        Directions.shared.calculate(options) { _, routes, _ in
+            guard let route = routes?.first else { return }
+            
+            self.route = route
+            
+            let simulatedLocationManager = SimulatedLocationManager(route: route)
+            simulatedLocationManager.speedMultiplier = 2
+            
+            self.viewController.startNavigation(with: route, locationManager: simulatedLocationManager)
+        }
+    }
+    
+    @objc
+    func cameraButtonTapped() {
+        guard let waypoint = self.waypoints.randomElement() else { return }
+        
+        func randomCLLocationDistance(min: CLLocationDistance, max: CLLocationDistance) -> CLLocationDistance {
+            CLLocationDistance(arc4random_uniform(UInt32(max - min)) + UInt32(min))
```

Comment:
Might as well address these lints for new code.

Reply:
Sadly SwiftLint is not running in Xcode, only on CI so I was not aware of those warnings